config/v1/feature_gates.go

```diff
@@ -0,0 +1,174 @@
+package v1
+
+type FeatureGateDescription struct {
```

Comment:
This is internal and not supposed to be exposed as an API right? Wondering if we should include a godoc explaining that this should not be included in/exposed via a CRD

Reply:
>This is internal and not supposed to be exposed as an API right? Wondering if we should include a godoc explaining that this should not be included in/exposed via a CRD

Not exposed via an API, but expose via golang to allow cluster-config-operator to learn it.